### PR TITLE
Rename cnl file to cml & cmlv

### PR DESCRIPTION
**Pull request by jinjhuli**
Rename file with "cnl" to "cml" & "cmlv".
Rename include file's name in related file.

Signed-off-by: jinjhuli <jin.jhu.lim@intel.com>

### CODE REVIEW

**Review by mauricema — APPROVED** [review]
Looks good to me.

File: Platform/CometlakeBoardPkg/AcpiTables/Dsdt/PchSerialIo.asl [hunk 1]
```diff
@@ -16,7 +16,7 @@
 // Include PchSsrialIo ACPI macros
 //
 // PCH_CNL - Default
-Include("PchSerialIoAcpiDefinesCnl.h")
+Include("PchSerialIoAcpiDefinesCml.h")
```
Comment (myguitar) [inline Platform/CometlakeBoardPkg/AcpiTables/Dsdt/PchSerialIo.asl:19]: Can you update PCH_CNL comment as well?

File: Platform/CometlakevBoardPkg/AcpiTables/Dsdt/PchSerialIo.asl [hunk 2]
```diff
@@ -16,7 +16,7 @@
 // Include PchSsrialIo ACPI macros
 //
 // PCH_CNL - Default
-Include("PchSerialIoAcpiDefinesCnl.h")
+Include("PchSerialIoAcpiDefinesCmlv.h")
```
Comment (myguitar) [inline Platform/CometlakevBoardPkg/AcpiTables/Dsdt/PchSerialIo.asl:19]: Can you update PCH_CNL comment as well?

**Review by myguitar — APPROVED** [review]
Can you update PCH_CNL comment as well? Other than that, it looks good to me.

File: Platform/CometlakeBoardPkg/AcpiTables/Dsdt/Gpio.asl [hunk 3]
```diff
@@ -6,7 +6,7 @@
 
 **/
 // PCH_CNL - Default
-Include("GpioAcpiDefinesCnl.h")
+Include("GpioAcpiDefinesCml.h")
```
[the remark this answers is not in the document]
Comment (jinjhuli) [reply Platform/CometlakeBoardPkg/AcpiTables/Dsdt/Gpio.asl:9]: Done :)

File: Platform/CometlakeBoardPkg/AcpiTables/Dsdt/PchSerialIo.asl (hunk 1 above)
Comment (jinjhuli) [reply Platform/CometlakeBoardPkg/AcpiTables/Dsdt/PchSerialIo.asl:19]: Done :)

File: Platform/CometlakevBoardPkg/AcpiTables/Dsdt/PchSerialIo.asl (hunk 2 above)
Comment (jinjhuli) [reply Platform/CometlakevBoardPkg/AcpiTables/Dsdt/PchSerialIo.asl:19]: Done :)

**Pull request merged**